Dynamically generate configs at run time using baseline-browser-mapping (#4)
web-platform-dx/browserslist-config-baseline: README.md

A [browserslist](https://www.npmjs.com/package/browserslist) config based on [Web Platform Baseline](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility).

Baseline identifies web platform features that work across browsers. Baseline helps you decide when to use a feature by telling you when it is less likely to cause compatibility problems for your site's visitors. There are two ways of targeting a Baseline feature set:

1. [Baseline Widely Available](#baseline-widely-available) - includes all features that were fully supported in Chrome, Edge, Firefox and Safari **30 months before the current date**.
2. [Baseline year feature sets](#baseline-year-feature-sets) - includes all features that were fully supported in Chrome, Edge, Firefox and Safari **at the end of a given calendar year**.

You should check your analytics to see which browser versions are prevalent in your userbase before selecting a Baseline target.

## Installation

To add `browserslist-config-baseline` to your project, use one of the following commands:

> ```sh
> # NPM
> npm i browserslist-config-baseline
> # yarn
> yarn add browserslist-config-baseline
> # If using bun, please run: \n' +
> bun add browserslist-config-baseline
> ```

Reviewer: ```suggestion
```sh
# npm
npm i browserslist-config-baseline
# yarn
yarn add browserslist-config-baseline
# bun
bun add browserslist-config-baseline
```
```

}
```

The minimum browser set for Baseline Widely Available changes frequently. Consider updating this module regularly by adding one of these commands to your build scripts:

```sh

> # NPM

Reviewer: ```suggestion
# npm
```

npm i browserslist-config-baseline@latest

# yarn
yarn upgrade --latest browserslist-config-baseline

# bun
bun update browserslist-config-baseline@latest
```

> If you haven't updated `browserslist-config-basline` in the last month and your installed version is lower than the latest published version, you will receive a console warning every time `browserslist` references this module.

Reviewer: Maybe something like this?

```suggestion
If you haven't updated `browserslist-config-basline` for a month or longer, it will check for updates when you run a Browserslist-compatible tool, and prompt you to upgrade.
```

Author: I'm going with:

> If you haven't updated `browserslist-config-basline` in the last month and you are targeting Widely Available, the package will check for updates whenever you run a browserslist-compatible tool and prompt you to upgrade if there is a new version of `browserslist-config-baseline` available.

We should add dependabot and have it bump the patch version and release every time there is an update to `web-features`, `@mdn/browser-compat-data` or `baseline-browser-mapping` as all three data sources are used in this module.
